python/doc/examples/reliability_sensitivity/reliability/plot_stiffened_panel.py

```diff
@@ -0,0 +1,143 @@
+"""
+Estimate a buckling probability
+===============================
+"""
+# %%
+#
+# In this example, we estimate the probability that the output of a function
+# exceeds a given threshold with the FORM method, the SORM method and an advanced
+# sampling method.
+
+# We consider the :ref:`stiffened panel model <use-case-stiffened-panel>`.
```

Comment:
```suggestion
# sampling method.
#
# We consider the :ref:`stiffened panel model <use-case-stiffened-panel>`.
```

Whenever there is an empty line in a cell, Sphinx switches from RST to Python, which leads to poor formatting.
![image](https://github.com/openturns/openturns/assets/31989332/3d15bc23-d592-490e-8073-624243ed1380)